gulpfile.js

```diff
@@ -273,4 +273,4 @@ gulp.task("release", ["common"], function (done) {
 });
 
 //default task is release
```

Comment:
change this comment to "default task is common"

Reply:
@r0h4n ack